COCOAPODS.md

```diff
@@ -96,39 +94,62 @@ the build process of an Xcode project.
 
 ## Add dependencies on Pod libraries
 
-You can add dependencies between a Kotlin project and Pod libraries [stored in the CocoaPods repository](#add-a-dependency-on-a-pod-library-from-the-cocoapods-repository) 
-and [stored locally](#add-a-dependency-on-a-pod-library-stored-locally).
+To add dependencies between a Kotlin project and a Pod library, you should [complete the initial configuration](#install-the-cocoapods-dependency-manager-and-plugin).
+This allows you to add dependencies on the following types of Pod libraries: 
+ * [A Pod library from a CocoaPods repository](#add-a-dependency-on-a-pod-library-from-the-cocoapods-repository) 
+ * [A Pod library stored locally](#add-a-dependency-on-a-pod-library-stored-locally)
+ * [A Pod library from a Git repository](#add-a-dependency-on-a-pod-library-from-the-git-repository)
+ * [A Pod library from an archive](#add-a-dependency-on-a-pod-library-from-an-archive)
+ * [A Pod library from a custom Spec repository](#add-a-dependency-on-a-pod-library-from-a-custom-spec-repository)
+ * [A Pod library with custom C Interop options](#add-a-dependency-on-a-pod-library-with-custom-c-interoperability-options)
+ * [A static Pod library](#add-a-dependency-on-a-static-pod-library)
```

Comment:
These last two sections seem to be orthogonal to the previous ones. However, the way this list is organized here and the contents of these sections don't help to understand that.

Reply:
I planned to restructure after the upcoming release: https://youtrack.jetbrains.com/issue/KT-43363